### ISSUE TITLE
quick question about replays

### ISSUE THREAD

**Issue by vagarwala** [bouwkast/MLBot#1]
Hi - 
I have a quick question: I downloaded your bot and it seems to work fine when trained on minireplays.  I tried to train it on a bigger set of replays. but received this error:

<img width="446" alt="screen shot 2017-02-05 at 12 36 35 pm" src="https://cloud.githubusercontent.com/assets/12286367/22628315/cabbff06-eb9f-11e6-9874-4f696dd27a72.png">

Could you explain what this error is & how I could fix it?  How did you train on more samples?

**Comment by bouwkast** [comment]
I just pushed an update to train_bot.py

Around lines 90-100 there is code for training on a single specific bot, originally it was training on whoever had the highest strength piece at the end of the game.

I commented those lines out (90 - 100) and un-commented lines 109 to 111, so now it should train on the winner and should work on any data set. 

Let me know if that doesn't solve your issue, but I'm pretty sure it should.
